src/bindings/python/nixl-meta/nixl/__init__.py

```
@@ -17,20 +17,38 @@
+
+def _get_torch_cuda_major() -> int | None:
+    """Return the CUDA major version that torch was built for, or None."""
     try:
-        _pkg = importlib.import_module(pkg)
-        break
+        from torch.version import cuda as _torch_cuda_ver
+
+        if _torch_cuda_ver is not None:
+            return int(_torch_cuda_ver.split(".")[0])
+    except Exception:
+        pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)